libminifi/include/Connection.h

```diff
@@ -180,7 +184,8 @@ class Connection
 	std::atomic<uint64_t> _maxQueueDataSize;
 	//! Flow File Expiration Duration in= MilliSeconds
 	std::atomic<uint64_t> _expiredDuration;
-
+	//! UUID string
+	std::string _uuidStr;
```

Comment:
We're making a transition to the google code style. I don't think consistency in a class is a concern now since we're in such flux. Doing that will help me avoid having to do new members.